pkg/controller/main-controller.go

```diff
@@ -410,15 +418,20 @@ func leaderRun(ctx context.Context, c *Controller, threadiness int, stopCh <-cha
 	if isOperatorConsoleTLS() {
 		klog.Info("Waiting for Console TLS")
 		go func() {
-			klog.Infof("Console TLS is enabled, starting console TLS certificate setup")
-			err := c.recreateOperatorConsoleCertsIfRequired(ctx)
-			if err != nil {
-				panic(err)
-			}
-			klog.Infof("Restarting Console pods")
-			err = c.rolloutRestartDeployment(getConsoleDeploymentName())
-			if err != nil {
-				klog.Errorf("Console deployment didn't restart: %s", err)
+			if utils.GetOperatorRuntime() == common.OperatorRuntimeOpenshift {
+				klog.Infof("Console TLS is enabled, skipping TLS certificate generation")
```

Comment:
I feel message should clarify reason for skipping cert generation as `Console TLS is enabled, skipping TLS certificate generation due to OpenShift based deployment`

Reply:
applied suggestion